src/averageAges.js

```diff
@@ -54,8 +72,29 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
+function asMother(people, child) {
+  return people.some(person => person.name === child.mother);
+}
+
+function motherBorn(people, mother) {
+  return people.find(person => person.name === mother).born;
```

Comment:
if the `person` does not exist, the `find` method returns `undefined`. `undefined.born` will trow an error.